packages/core/src/services/chains/ChainValidator/index.ts

```diff
@@ -206,7 +224,7 @@ export const validateChain = async (
       ...conversation,
       messages: rule?.messages ?? conversation.messages,
     },
-    schema: getInputSchema(chainCompleted, config, configOverrides),
-    output: getOutputType(chainCompleted, config, configOverrides),
+    schema,
+    output,
   })
 }
```

Comment:
not sure if there are tests for chain validator but if there are you know what to do 👌🏼

Reply:
There is not :(